src/components/MyAccount/Settings/PasswordModal.tsx

```diff
@@ -41,7 +42,11 @@ const PasswordModal = ({ patron }: { patron: Patron }) => {
           </li>
           <li> PINs and PASSWORDS must NOT contain a period.</li>
         </List>
-        <PasswordChangeForm patron={patron} updateModal={updateModal} />
+        <PasswordChangeForm
+          patron={patron}
+          updateModal={updateModal}
+          updateModalToLoading={() => setModalProps(loadingProps)}
```

Comment:
Can you rename this to onModalSubmit or something like that to indicate that it's a callback for a specific action (as opposed to a description of what's being passed into the callback)